exp-models/addon/models/account.js

```diff
@@ -2,14 +2,34 @@
 Manage data about one or more documents in the accounts collection
  */
 
+import moment from 'moment';
+
+import Ember from 'ember';
 import DS from 'ember-data';
 import JamModel from '../mixins/jam-model';
 
+var Profile = Ember.Object.extend({
+    birthday: null,
+    age: Ember.computed('birthday', function() {
+        var bd = moment(this.get('birthday'));
+        return moment(new Date()).diff(bd, 'days');
+    })
+});
+
 export default DS.Model.extend(JamModel, {
     username: DS.attr('string'),
     password: DS.attr('string'),
-    profiles: DS.attr(),  // TODO: Pages using profiles will be fragile unless there is a clear, explicit contract of what fields ember can always expect to be present. This nested document doesn't inherently specify as such.
-
+    _profiles: DS.attr(), // TODO: Pages using profiles will be fragile unless there is a clear, explicit contract of what fields ember can always expect to be present. This nested document doesn't inherently specify as such.
+    profiles: Ember.computed('_profiles', {
```

Comment:
Come to think of it- is there any reason we can't just use a custom `DS.Transform` in place of the computed field?

You'd want to test it using `createRecord` (to ensure that something that is already a plain object  would serialize correctly). The idea would be to define the field as: `profiles = DS.attr('customprofiletransformpowersactivate'`)
http://emberjs.com/api/data/classes/DS.Transform.html